benchmark/main.go

```diff
@@ -72,6 +73,8 @@ func main() {
 		filter = iptables.New()
 	case "ipset":
 		filter = ipset.New()
+	case "calico":
+		filter = calico.New(nets, iface)
```

Comment:
Because I am splitting the generated nets in the batches of 10K, for large number of nets it was becoming a time cnsuming operation espcially when the same thing has to happen for creating and deleting of the calico resources.